### PR TITLE
Add ProsperWorks data source

### PR DESCRIPTION
**Pull request by kaftz**

### CODE REVIEW

**Comment by kaftz** [comment]
I signed it!

**Comment by kaftz** [comment]
I signed it!

**Comment by pfrisella** [comment]
Can you also review and resolve the CLA issue.
If you want to check which email is associated with the commits you can view the patch file by adding .patch to the pull request URL: https://github.com/googledatastudio/ds-data-registry/pull/66.patch

**Comment by kaftz** [comment]
I signed it!

**Comment by kaftz** [comment]
@pfrisella - looks like it just went through a few minutes ago.

File: sources/prosperworks.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "PROSPERWORKS",
+  "name": "ProsperWorks",
+  "categories": ["ANALYTICS"],
```
Comment (pfrisella) [inline sources/prosperworks.json]: is there another more appropriate category that would be helpful for users to know and find this source? If a category doesn't exist you can add it to https://github.com/googledatastudio/ds-data-registry/blob/master/categories.json

Comment (kaftz) [reply sources/prosperworks.json]: Yeah, I can add a better one.

**Comment by pfrisella** [comment]
LGTM. Thanks for making the updates and contributing!

**Pull request merged**